spec/requests/articles/articles_update_spec.rb

```diff
@@ -4,7 +4,11 @@
   let(:organization) { create(:organization) }
   let(:organization2) { create(:organization) }
   let(:user) { create(:user, :org_admin) }
-  let(:user2) { create(:user, organization_id: organization2.id) }
+  let(:user2) do
```

Comment:
What do you think about creating a factory that would create a user with org membership and use in instead creating a user + a membership inside a `let`?

Reply:
I almost thought about it, I checked 

https://github.com/thepracticaldev/dev.to/blob/2dbb558b30c15b2b77a7d67c41f8c9a303567e1f/spec/factories/users.rb#L83-L88

but I don't know if traits can have arguments, I was too lazy to go search factorybot's documentation and let that slide :)